### PR TITLE
Lab5

### PR DESCRIPTION
**Pull request by bkmceh**
@ezio-42, @danmory u r welcome

### CODE REVIEW

File: terraform/yandex-cloud-terraform/main.tf [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+  resources {
+    cores  = var.app_server_cores
+    memory = var.app_server_mem
+  }
+
+  boot_disk {
+    initialize_params {
+      image_id = var.image_id_name # Ubuntu 22.04
+      size     = 10
```
Comment (ezio-42) [inline terraform/yandex-cloud-terraform/main.tf]: keep size as var

File: ansible/out.txt [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (ezio-42) [inline ansible/out.txt]: I think this file could be deleted

Comment (bkmceh) [reply ansible/out.txt]: opsy sorry

File: ansible/playbooks/dev/main.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+- name: install docker
+  hosts: managed_nodes
```
Comment (danmory) [inline ansible/playbooks/dev/main.yml:2]: why not "yc_host" to be more specific and to avoid installing docker on hosts unnecessarily?

**Comment by danmory** [comment]
and I advise you to finally resolve the conflicts with target branch :)
image

**Pull request closed without merging**